src/cryptonote_config.h

```diff
@@ -93,11 +93,15 @@
 #define DIFFICULTY_BLOCKS_ESTIMATE_TIMESPAN             DIFFICULTY_TARGET_V1 //just alias; used by tests
 
 
-#define BLOCKS_IDS_SYNCHRONIZING_DEFAULT_COUNT          10000  //by default, blocks ids count in synchronizing
-#define BLOCKS_IDS_SYNCHRONIZING_MAX_COUNT              25000  //max blocks ids count in synchronizing
-#define BLOCKS_SYNCHRONIZING_DEFAULT_COUNT_PRE_V4       100    //by default, blocks count in blocks downloading
```

Comment:
Are these whitespace changes?

Reply:
The problem is name of `BLOCKS_SYNCHRONIZING_DEFAULT_MEDIAN_WINDOW` is too long. 

For now lets just test functionality. I will fix these when submitting to `monero-project/monero`.